src/CMakeLists.txt

```diff
@@ -130,12 +151,29 @@ else()
     target_sources(umf PRIVATE memory_pool_default.c)
 endif()
 
+# Currently, Level Zero provider is only built for Linux
+if(LINUX AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
```

Comment:
I believe when resolving merge conflicts you omitted one change - this `if` is now duplicated - please see a few lines above ( https://github.com/oneapi-src/unified-memory-framework/pull/297/files#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4R100-R107 )

Reply:
Right - this file changes very often :) removed